some path adaption to gentoo vdr
use corefonts instead of ttf-bitstream-vera, #335782

Signed-of-by: Joerg Bornkessel <hd_brummy@gentoo.org> (1 Mar 2015)
diff -Naur burn-0.2.0.orig/burn.c burn-0.2.0/burn.c
--- burn-0.2.0.orig/burn.c	2012-02-12 19:46:51.000000000 +0100
+++ burn-0.2.0/burn.c	2012-02-12 19:49:21.000000000 +0100
@@ -96,11 +96,11 @@
 
 	bool plugin::Start()
 	{
-		m_configPath = ConfigDirectory(PLUGIN_NAME);
+		m_configPath = "/usr/share/vdr/burn";
 
 		manager::start();
 		logger_vdr::start();
-		gdwrapper::setup::set_font_path( m_configPath + "/fonts" );
+		gdwrapper::setup::set_font_path( "/usr/share/fonts/corefonts" );
 		if ( !skin_list::get().load( get_config_path() ) )
 			return false;
 
diff -Naur burn-0.2.0.orig/vdrburn-dvd.sh burn-0.2.0/vdrburn-dvd.sh
--- burn-0.2.0.orig/vdrburn-dvd.sh	2012-02-12 19:46:51.000000000 +0100
+++ burn-0.2.0/vdrburn-dvd.sh	2012-02-12 19:52:22.000000000 +0100
@@ -92,9 +92,8 @@
 			CUT="-cut $MPEG_DATA_PATH/px.cut"
 		fi
 
-		$IO_NICE $JAVA_EXE -Djava.awt.headless=true \
-				-jar $PROJECTX_JAR \
-				-ini $CONFIG_PATH/ProjectX.ini \
+				/usr/bin/projectx_cli \
+				-ini /usr/share/vdr/burn/projectx-vdr.ini \
 				$TTXT_OPTS \
 				$CUT -id $USED_TRACKS \
 				-demux -out "$MPEG_DATA_PATH" -name vdrsync \
diff -Naur burn-0.2.0.orig/vdrburn-dvd.sh burn-0.2.0/vdrburn-dvd.sh
--- burn-0.2.0.orig/vdrburn-dvd.sh	2012-02-15 21:58:34.000000000 +0100
+++ burn-0.2.0/vdrburn-dvd.sh	2012-02-15 22:00:47.000000000 +0100
@@ -78,7 +78,7 @@
 	;;
 
 	demux)
-		[ -r $CONFIG_PATH/vdrburn-dvd.conf ] && . $CONFIG_PATH/vdrburn-dvd.conf
+		PROJECTX_JAR="/usr/share/projectx/lib/projectx.jar"
 		[ -z "$JAVA_EXE" ] && JAVA_EXE=$(which java)
 
 		[ ! -x "$JAVA_EXE" ] && echo "ERROR: java executable \"$JAVA_EXE\" not found" && exit 1
